tests/behat/backup_restore.feature

```diff
@@ -0,0 +1,74 @@
+@mod @mod_surveypro @current
+Feature: Duplicate a surveyspro instance
+  In order to validate backup and restore process
+  As a teacher
+  I backup and restore a surveypro instance.
+
+  Background:
+    Given the following "courses" exist:
+      | fullname | shortname | category | numsections |
+      | Course 1 | C1        | 0        | 3           |
+    And I log in as "admin"
+
+  @javascript
+  Scenario: Duplicate a surveypro with some item
+    Given the following "activities" exist:
+      | activity   | name           | intro              | course | idnumber   |
+      | surveypro  | Surveypro test | For testing backup | C1     | surveypro1 |
+    And surveypro "surveypro test" contains the following items:
+      | type   | plugin      |
+      | format | label       |
+      | format | fieldset    |
+      | field  | age         |
+      | field  | autofill    |
+      | field  | boolean     |
+      | field  | character   |
+      | field  | checkbox    |
+      | field  | date        |
+      | field  | datetime    |
+      | field  | fileupload  |
+      | field  | integer     |
+      | field  | multiselect |
+      | field  | numeric     |
+      | format | fieldsetend |
+      | field  | radiobutton |
+      | format | pagebreak   |
+      | field  | rate        |
+      | field  | recurrence  |
+      | field  | select      |
+      | field  | shortdate   |
+      | field  | textarea    |
+      | field  | time        |
+
+    When I backup "Course 1" course using this options:
+      | Confirmation | Filename | test_backup.mbz |
+    Then I should see "Restore"
+
+    And I restore "test_backup.mbz" backup into a new course using this options:
+      | Schema | Course name       | Copy of course 1 |
+      | Schema | Course short name | C2               |
+    Then I should see "Copy of course 1"
+
+    And I follow "Surveypro test"
+    And I follow "Layout"
+
+    Then I should see "Welcome to this new instance of surveypro"
+    And I should see "Grouped data inside"
+    And I should see "How old were you at you first access to narcotic?"
```

Comment:
hehe, I've seen you're using this here and there... maybe it would be better to ask "How old were you the first time you ate pizza" (more innocent question, just in case). :-)